test/functional/services/common/find.ts

```diff
@@ -129,7 +129,9 @@ export class FindService extends FtrService {
     timeout: number = this.WAIT_FOR_EXISTS_TIME
   ): Promise<boolean> {
     this.log.debug(`Find.descendantExistsByCssSelector('${selector}') with timeout=${timeout}`);
+    await this._withTimeout(timeout);
     const els = await parentElement._webElement.findElements(By.css(selector));
+    await this._withTimeout(this.defaultFindTimeout);
```

Comment:
I believe this needs to be in a `finally` so that if the `findElements` errors or hits a timeout, the timeout will get reverted